sycl/doc/design/SharedLibraries.md

```diff
@@ -160,8 +160,9 @@ if the Function "can be imported".  A `canBeImportedFunction` is:
 
   1. Not an intrinsic
   2. Name does not start with "__"
-  3. Demangled name does not start with "__"
-  4. Must be a `SYCL_EXTERNAL` function
+  3. Is not a SPIRV, SYCL, or ESIMD builtin function
```

Comment:
I suspect this document change is not related to the code change. right? Just wanted to make sure I understand this. Thanks

Reply:
The new point 3 is not related to the changes in this PR.   It is documenting a change that was previously done.